mailpoet/lib/Form/DisplayFormInWPContent.php

```diff
@@ -117,12 +117,11 @@ private function shouldDisplay(): bool {
   private function displayFormInListingPage(): bool {
     $displayCheck = $this->wp->applyFilters('mailpoet_display_form_in_product_listing', true);
 
-    if (function_exists('wc_get_page_id')) {
-      $shopPageId = wc_get_page_id('shop');
-      $this->wooShopPageId = $shopPageId && $shopPageId >= 0 ? $shopPageId : null;
-      if ($displayCheck && !is_null($this->wooShopPageId) && $this->wp->isPage($shopPageId)) {
-        return true;
-      }
+    $shopPageId = $this->wp->wcGetPageId('shop');
```

Comment:
We have [a WooCommerce Helper class](https://github.com/mailpoet/mailpoet/blob/trunk/mailpoet/lib/WooCommerce/Helper.php) that is used the same way as WPFunctions. Could you please move the wrapped function into the Woo helper so that it is together with other Woo funcitons?

Reply:
Fixed in https://github.com/mailpoet/mailpoet/pull/4459/commits/d51ae1859d890471ba4efd3529380d10fa45ce37